scss/components/_cookies-banner.scss

```diff
@@ -68,4 +68,17 @@
         padding: 0;
         margin: 8px 0;
     }
+
+    &__preferences-success {
+        display: block;
+        border-left: 7px solid $salem;
+        background: $lily-white;
+    }
+
+    &__preferences-success-body {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        padding: 8px 0 8px 16px;
```

Comment:
This should be using $col for horizontal sizes and $baseline for verticle... I think.